### ISSUE TITLE
atuin init fails in nushell

### ISSUE THREAD

**Issue by Nezteb** [atuinsh/atuin#1282]
**Describe the bug**

Following [the `atuin` docs for `nushell`](https://github.com/atuinsh/atuin#nushell) gives me an error suggesting the syntax is invalid.

**To Reproduce**

Put these in your `config.nu` file:

```bash
atuin init nu | save ~/.local/share/atuin/init.nu
source ~/.local/share/atuin/init.nu
```

This gives an error:

```bash
Error: nu::parser::expected_keyword

  × Expected keyword.
   ╭─[/Users/noah/Library/Application Support/nushell/custom/core.nu:2:1]
 2 │
 3 │ source ~/.local/share/atuin/init.nu
   · ───┬──
   ·    ╰── expected def, const, def-env, extern, extern-wrapped, alias, use, module, export or export-env keyword
 4 │
   ╰────
```

I also tried switching to the `use` keyword, but that gives a different error:
```bash
Error: nu::parser::expected_keyword

  × Expected keyword.
   ╭─[/Users/noah/.local/share/atuin/init.nu:1:1]
 1 │ # Source this in your ~/.config/nushell/config.nu
 2 │ $env.ATUIN_SESSION = (atuin uuid)
   · ─────────┬────────
   ·          ╰── expected def, const, def-env, extern, extern-wrapped, alias, use, module, export or export-env keyword
 3 │
   ╰────
```

**Expected behavior**

Ideally this should work.

**Device (please complete the following information):**

- macOS Ventura 13.6 (Apple M2 Pro)
- nushell 0.84.0
- atuin 16.0.0

**Comment by ellie** [comment]
I'm not sure what's going on here because supposedly `source` is a valid keyword

https://www.nushell.sh/commands/docs/source.html

**Comment by Nezteb** [comment]
Yeah I'm also confused. I figured I'd report it just in case others notice this, but it still could be just a "my machine" thing.

I'll do some experimenting and see if I can figure it out!

**Comment by Nezteb** [comment]
This was a me problem.

`env.nu` loads first, `config.nu` loads next. I split out some stuff from `config.nu` into a separate `custom.nu` file for my own tweaks. I set it up to be imported like `use custom.nu` based on docs. In that file, I put the `source ~/.local/share/atuin/init.nu` line. This is bad because everything in `custom.nu` is imported as a module and not a script.

tl;dr Be careful with mixing and matching scriptsm/modules/overlays. Don't `use` a file that calls `source` on other files. 

I'm all good now!
